plugin/trino-loki/src/main/java/io/trino/plugin/loki/model/package-info.java

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
Which parts of the code using this class ?

Reply:
Non. The linter complained and had me add this.